doc/project/technical_committee.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "OpenTitan Technical Committee"
+---
+
+The OpenTitan Technical Committee is part of the [OpenTitan governance structure]({{< relref "doc/project" >}}) and the definitive description of its operations and responsibilities are stated in the [OpenTitan Technical Charter](https://static.opentitan.org/technical-charter.pdf).
+Key responsibilities defined in that document include:
+* Oversight of the [RFC process]({{< relref "rfc_process" >}}), including voting to approve, reject, postpone or revise RFCs.
+* Creating [Commit Escalation Guidelines]({{< relref "committers" >}}) setting out when Committers should escalate a particular Commit to the TC (e.g. for cross-cutting changes), and when Committers should recommend a design rationale write-up or RFC for wider feedback.
+* Maintaining the list of [committers]({{< relref "committers" >}}).
+
+Expectations of Technical Committee members include the following:
+* Be active participants in development discussions within the OpenTitan project, beyond just Technical Committee meetings.
+* Consider comments and feedback that have been made on proposals during the project-wide review process and ensure any concerns are adequately considered and addressed.
+* Ensure that feedback has been sought from all relevant stakeholders, and appropriate effort has been made to find consensus within the OpenTitan contributor community.
+* Put aside sufficient time on a week-by-week basis to study and feedback on active proposals.
```

Comment:
```suggestion
* Put aside sufficient time on a week-by-week basis to study active proposals and give feedback.
```
Not sure whether "to feedback" exists, if so, just ignore

Reply:
That's a better phrasing, thanks.